.envrc

```diff
@@ -21,7 +21,7 @@ SENTRY_ROOT="$(
     pwd -P
 )"
 
-source "${SENTRY_ROOT}/scripts/lib.sh"
+source "${SENTRY_ROOT}/"scripts/lib.sh
```

Comment:
this is unrelated to your change.  please separate unrelated things to separate patches.  also this change "improves nothing" -- I'd even argue the position of the closing quote containing the slash makes it worse